lib/modules/manager/terraform/extractors/resources/helm-release.ts

```diff
@@ -27,7 +27,13 @@ export class HelmReleaseExtractor extends DependencyExtractor {
         datasource: HelmDatasource.id,
       };
       if (is.nonEmptyString(helmRelease.repository)) {
-        dep.registryUrls = [helmRelease.repository];
+        if (isOCIRegistry(helmRelease.repository)) {
+          // For oci repos, we remove the oci:// and use the docker datasource
+          dep.registryUrls = [helmRelease.repository.replace('oci://', '')];
```

Comment:
```suggestion
          dep.registryUrls = [helmRelease.repository.replace('oci://', 'https://')];
```
i think we need https here

Reply:
@viceice Done. I actually went without `https://` due to the fact that [it seems that https is added automatically anyway](https://github.com/jalaziz/renovate/blob/e096105b14601f84607845af9ced1fa556466ecb/lib/modules/datasource/docker/index.ts#L273-L280).